fio.1

```diff
@@ -344,27 +344,60 @@ Sequential reads.
 .B write
 Sequential writes.
 .TP
-.B trim
-Sequential trims (Linux block devices only).
-.TP
 .B randread
 Random reads.
 .TP
 .B randwrite
 Random writes.
 .TP
-.B randtrim
-Random trims (Linux block devices only).
-.TP
 .B rw, readwrite
-Mixed sequential reads and writes.
+Mixed sequential read or write.
 .TP
 .B randrw
-Mixed random reads and writes.
+Mixed random read or write.
+.RE
+.RS
+.P
+Trim I/O has several requirements:
+.RS
+ - File system and OS support varies but Linux block devices accept trims.
+   You need privilege to write to a Linux block device.
+   See example fio: track-mem.fio
+ - Often minimal block size required. Linux on VMware requires 1 MB
+ - VMware requires minimum VM OS hardware level 11
+ - To verify the trim I/Os requires verify_track
+.RE
```

Comment:
I don't know you can say much more about the requirements other than "you may need extra privileges and your OS/Storage may enforce a minimum trim block size along with other constraints" e.g. I thought with ESXi 6.0 you had to also set EnableBlockDelete on the VM for UNMAP commands to work (obviously this has changed with ESXi 6.5)?